Jenkinsfile

```diff
@@ -8,7 +8,7 @@ def initializeEnvironment() {
   env.GITHUB_BRANCH_URL = "${GITHUB_PROJECT_URL}/tree/${env.BRANCH_NAME}"
   env.GITHUB_COMMIT_URL = "${GITHUB_PROJECT_URL}/commit/${env.GIT_COMMIT}"
 
-  env.MAVEN_HOME = "${env.HOME}/.mvn/apache-maven-3.6.0"
+  env.MAVEN_HOME = "${env.HOME}/.mvn/apache-maven-3.6.3"
```

Comment:
This change assumes that we're using a Java runner which includes [the addition of Maven 3.6.3](https://github.com/riptano/openstack-jenkins-drivers/pull/102)